test/Cpp/src/multiport/FullyConnectedAdressableAfter.lf

```diff
@@ -0,0 +1,15 @@
+// In this pattern, each node can send direct messages to individual other nodes
```

Comment:
The file name has a spelling error: FullyConnectedAdressableAfter -> FullyConnectedAddressableAfter